### ISSUE TITLE
Running this command "    >pip install face-alignment==1.0.0 msgpack_numpy pyyaml==5.1 --use-feature==2020-resolver    " trying to push avatarify to google collab and got the below error message

### ISSUE THREAD

**Issue by Elektrobotcdp** [alievk/avatarify-python#645]
**Describe the bug**

**To Reproduce**

**Info (please complete the following information):**
 - OS (e.g., Linux):  Windows
 - GPU model: 

 - Any other relevant information: 

**Screenshots**

**Logs**

**Comment by Elektrobotcdp** [comment]
Yes, it doesn't feels right, that's why i reported the bug @JohanAR

**Comment by JohanAR** [comment]
You haven't reported anything. You've created an empty issue report with some word jumble in the title. It does not describe the error or provide any info about what you did before the error happened.

**Comment by Skyhigh990** [comment]
pip install [options] <requirement specifier> [package-index-options] . pip install [options] -r «requirements file> [package-index-options] pip install [options] [-e] ‹vs project url> pip install [options] [-e] ‹local project path> pip install [options] <archive url/path> option --use-feature: invalid choice: '2020-resolver' (choose from 'fast-deps', "truststore', 'n o-binary-enable-wheel-cache') avatarify    Please i need help as am going through the  same thing have instal alot of things i saw online even paid someone who didn't even do it right ....@alievk ...I will appreciate your help on this

**Comment by Skyhigh990** [comment]
Usage:
  pip install [options] <requirement specifier> [package-index-options] ...
  pip install [options] -r <requirements file> [package-index-options] ...
  pip install [options] [-e] <vcs project url> ...
  pip install [options] [-e] <local project path> ...
  pip install [options] <archive url/path> ...

option --use-feature: invalid choice: '2020-resolver' (choose from 'fast-deps', 'truststore', 'no-binary-enable-wheel-cache')
